src/repositories/updater.rs

```diff
@@ -152,7 +152,19 @@ impl RepositoryStatsUpdater {
             // FIXME: The collect can be avoided if we use Itertools::chunks:
             // https://docs.rs/itertools/0.10.0/itertools/trait.Itertools.html#method.chunks.
             for chunk in needs_update.chunks(updater.chunk_size()) {
-                let res = updater.fetch_repositories(chunk)?;
+                let res = match updater.fetch_repositories(chunk) {
+                    Ok(r) => r,
+                    Err(err) => {
+                        if err.downcast_ref::<RateLimitReached>().is_some() {
+                            warn!(
+                                "rate limit reached, blocked the `{}` repository stats updater",
```

Comment:
"Blocked" doesn't seem correct, it will still collect stats the next time you run it. Maybe "ignored" is better?

Reply:
I took the original message. Maybe "stopped"?

Reply:
"skipping" maybe - that seems to imply it's temporary.

Reply:
Let's go for skipping then!